installer.sh

```diff
@@ -102,24 +102,72 @@ Set the OUT_DIR environment variable to change the installation directory:
 $ curl -fsSL $INSTALLER_URL | OUT_DIR=. bash
 
 EOF
+  if [ ! -d "${OUT_DIR}" ]; then
+    mkdir -p "$OUT_DIR"
+  fi
+
   if [ -w "${OUT_DIR}" ]; then
     read -p "Press enter to continue (or cancel with Ctrl+C):"
     mv "$TMP_DIR/$EXE" "$OUT_DIR"
   else
-    printf "Sudo is required to run \"sudo mv %s %s\":\n" "$TMP_DIR/$EXE" "$OUT_DIR"
-    sudo mv "$TMP_DIR/$EXE" "$OUT_DIR"
+    echo "$OUT_DIR is not writable."
+    exit 1
   fi
 fi
 
 rm -r "$TMP_DIR"
```

Comment:
should we remove the old path?

Reply:
I think the executable should stay in a writable space, and we should avoid the default install to `/local/usr/bin`.

Reply:
no definitely, I meant deleting today's default path, so next time people run the installer, they get everything correct?

Reply:
Oh, it should override the existing binary without any issue.